FluentUI.Demo/src/main/res/drawable/ic_fluent_link_24_regular.xml

```diff
@@ -10,7 +9,6 @@
     android:viewportWidth="24"
     android:viewportHeight="24">
     <path
-        android:fillColor="@color/fluent_default_icon_tint"
```

Comment:
Why was this replaced with the same value hardcoded

Reply:
Fixed it